### DISCUSSION TITLE
Automerge with Bitbucket Server and required reviewer approval

### DISCUSSION THREAD

**Discussion by biergit** [renovatebot/renovate#11709] [Request Help] [answered]
Hi
is it possible to use automerge with Bitbucket Server and required reviewer approval? If so how?

**Comment by viceice** [comment]
Not sure, maybe if you use `automergeType=branch`.

**Comment by biergit** [comment, marked as answer]
Hi @viceice 
that's probably the best option right now. I can configure Bitbucket to prevent pushes to certain integration branches except for the user that Renovate uses. Thanks!
